packages/contracts/test/voting/token-voting.ts

```diff
@@ -107,6 +116,28 @@ describe('TokenVoting', function () {
     );
   });
 
+  async function setBalances(
+    receivers: string[],
+    amounts: number[],
+    totalSupply: number
+  ) {
+    expect(receivers.length).to.equal(amounts.length);
+
+    let summedAmounts = amounts.reduce((a, b) => a + b, 0);
+
+    expect(summedAmounts).to.be.lte(totalSupply);
+
+    // Mint the difference between `totalSupply` and `summedAmounts` to `address(0)`
+    if (summedAmounts < totalSupply) {
+      receivers.push(`0x${'dead'.repeat(10)}`);
+      amounts.push(totalSupply - summedAmounts);
+    }
+
+    for (let i = 0; i < receivers.length; i++) {
```

Comment:
run that in a non blocking loop to speed things up:
```typescript
const promises = receivers.map((recv, i)=> governanceErc20Mock.setBalance(recv, amounts[i]))
await Promise.all(promises)
```

Reply:
Fixed in 08aa9e5c57d9581a9960d89c3329b258823f8637

Reply:
Improved in e66e59747b75c6ce535bde30be1342d74a358978